rasa/core/channels/rasa_chat.py

```diff
@@ -90,13 +90,22 @@ def _decode_jwt(self, bearer_token: Text) -> Dict:
     async def _extract_sender(self, req: Request) -> Optional[Text]:
         """Fetch user from the Rasa X Admin API"""
 
+        user = None
         if req.headers.get("Authorization"):
             user = await self._decode_bearer_token(req.headers["Authorization"])
-            if user:
-                return user["username"]
 
-        user = await self._decode_bearer_token(req.args.get("token", default=None))
+        if not user:
+            user = await self._decode_bearer_token(req.args.get("token", default=None))
```

Comment:
Rename this to something like `payload` or `jwt_payload` because it is misleading when labelled user (I hope our user doesn't have an expiry time for example!)